Tests/web3swiftTests/localTests/EthereumAddressTest.swift

```diff
@@ -35,4 +35,15 @@ class EthereumAddressTest: XCTestCase {
         XCTAssertNil(ethereumAddress)
     }
 
+    func testDescription() async throws {
+        let rawAddress = "0x200eb5ccda1c35b0f5bf82552fdd65a8aee98e79"
+        let ethereumAddress = EthereumAddress(rawAddress)
+
+        let sut = String(describing: ethereumAddress)
+        print(sut)
+
+        XCTAssertTrue(sut.contains("EthereumAddress\n"))
+        XCTAssertTrue(sut.contains("type: normal\n"))
+        XCTAssertTrue(sut.contains("address: 0x"))
```

Comment:
Why not `"address: 0x200eb5ccda1c35b0f5bf82552fdd65a8aee98e79"`?

Reply:
To be honest, some characters are in uppercase and others not, not clear to me why. 
I can do lowercase or uppercase but I didn't feel confortable doing that wihtout understanding it in detail

Reply:
This is what the log prints
"0x200EB5cCdA1c35B0F5Bf82552FDD65a8AEe98E79"
as you can see there are hex chars in upper/lowercase

Reply:
Added anyway